client/options.go

```diff
@@ -121,8 +121,9 @@ type SearchQueryOption struct {
 	Limit  int64
 	Offset int64
 
-	IgnoreGrowing bool
-	ForTuning     bool
+	IgnoreGrowing     bool
+	ForTuning         bool
+	RangeSearchParams map[string]float64
```

Comment:
Recommend change the value type to `any`, then it could add params with another types